frontend/src/components/eMIB/BackgroundSection.jsx

```diff
@@ -56,19 +54,21 @@ class BackgroundSection extends Component {
             const treeView = processTreeContent(contentItem[treeType], treeType);
 
             // Determine what image to use.
-            // Todo(caleybrock) - make this more dynamic in the future.
-            let imageSource;
+            // TODO(caleybrock) - move these images to a secure file server like S3.
+            // They are currently stored in the public folder which means
+            // you don't need to have any authentication to access org chart images.
+            let imageSource = `/orgCharts/${testNameId}/`;
             if (treeType === "organizational_structure_tree_child") {
               if (currentLanguage === "en") {
```

Comment:
I know that this is not part of your changes, but can we use the `LANGUAGES` variable here instead of `en`?
Or fix that in another branch?

Reply:
I'll fix in another PR since it's not related to this change.